tests/unit/Middleware/Data/DataMiddlewareCest.php

```diff
@@ -48,7 +48,7 @@ public function exceptionOnInvalidDataYml(\UnitTester $I)
         $I->seeExceptionThrown(InvalidSiteConfigException::class, function () {
             $fs = new FileSystem();
             $fs->createDirectory('/sites/www.example.com/data', true);
-            file_put_contents($fs->path('/sites/www.example.com/data/dataSet.yml'), '  Invalid yml content');
+            file_put_contents($fs->path('/sites/www.example.com/data/dataSet.yml'), '"Invalid yml content');
```

Comment:
It has  a meaning. The previous version was apparently a valid `yml` file and the test was failing. I guess the parser got updated and became more leniant.